docs/developer_manual/installation.rst

```diff
@@ -47,6 +47,57 @@ If you need to recompile::
     make clean
     make develop
 
+
+On Windows
+----------
+
+Microsoft Visual Studio is required for linking the Fortran binaries
+
+    * https://visualstudio.microsoft.com/downloads/
+    * During installation check the box to include :code:`Desktop development with C++`
+
+Intel oneAPI HPC toolkit is required for compiling the Fortran binaries (you do not need the base kit)
+
+    * https://www.intel.com/content/www/us/en/developer/tools/oneapi/hpc-toolkit-download.html
+    * Install to the default file location
+    
+Create a **"LIB"** environment variable to point towards the intel directory for compiler :code:`.lib` files
+
+    * If oneAPI is installed to the default location, assign the LIB user variable a value of:
+    
+        :code:`C:\Program Files (x86)\Intel\oneAPI\compiler\2022.1.0\windows\compiler\lib\intel64_win`
+    
+    * If oneAPI is installed to a different location then adjust the path above as necessary
+
+Test if your Fortran compiler was installed correctly by entering :code:`ifort` on your command line
+
+Open the anaconda powershell and create a new Python 3.9 environment (with numpy installed) for Capytaine-related development (e.g. :code:`capy_dev`)::
+    
+    conda create --name capy_dev python=3.9 numpy=1.20 scipy pip
+    conda activate capy_dev
+        
+Clone the Capytaine repo to your preferred location (e.g. "C:/code/")::
+        
+    cd C:/code/
+    git clone https://github.com/capytaine/capytaine.git
```

Comment:
Need `--recurse-submodules` because of the libDelhommeau submodule, as rightfully noticed by @nhu-vnguyen